gini/gini-vision-lib-android
# Fix layout references in XMLs

When using the AAR files compiled with the Android SDK, some view layout references weren't found because the view itself was declared below in the XML.
This PR also fixes an issue with gradle due to missing `aar` files when declaring `jcenter` repository before `google` repository inside the `build.gradle` file.

### How to test
Used the AAR files compiled with the Android SDK and check that those view layout references are found without problems.

### Merging
Automatic
